src/Forside/barnehage/vurderingstekster.tsx

```diff
@@ -22,10 +23,64 @@ export const getVurderingstekst = (
             return getVurderingstekstLangtid(sykefraværResultat, harBransje);
         case SammenligningsType.KORTTID:
             return getVurderingstekstKorttid(sykefraværResultat, harBransje);
+        case SammenligningsType.GRADERT: // TODO nesten ferdig, sjekk om ufullstendig data tekst er ok ?
```

Comment:
Er ikke teksten for ufullstendig data OK?

Reply:
Det er nok ok, tenkte bare en dobbeltsjekk